applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc

```diff
@@ -11,28 +11,29 @@ image::odc_add_view.png[Add View]
 
 * *From Git*: Use this option to import an existing codebase in a Git repository to create, build, and deploy an application on {product-title}.
 * *Container Image*: Use existing images from an image stream or registry to deploy it on to {product-title}.
-* *From Catalog*: Explore the *Developer Catalog* to select the required applications, services, or source to image builders and add it to your project.
 * *From Dockerfile*: Import a dockerfile from your Git repository to build and deploy an application.
 * *YAML*: Use the editor to add YAML or JSON definitions to create and modify resources.
+* *From Catalog*: Explore the *Developer Catalog* to select the required applications, services, or source to image builders and add it to your project.
 * *Database*: See the *Developer Catalog* to select the required database service and add it to your application.
+* *Operator Backed*: Explore the *Developer Catalog* to select and deploy the required Operator-managed service.
+* *Helm Chart*: Explore the *Developer Catalog* to select the required Helm chart to simplify deployment of applications and services.
 
 ifdef::openshift-enterprise,openshift-webscale[]
-[NOTE]
-====
-Serverless options in the *Developer* perspective are displayed only if the xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#serverless-install-web-console_installing-openshift-serverless[*OpenShift Serverless Operator*] is installed in your cluster.
-====
+
+Note that, certain options, such as *Pipelines*, *Event Source*, and *Import Virtual Machines* are displayed only when the xref:../../pipelines/installing-pipelines.adoc#op-installing-pipelines-operator-in-web-console_installing-pipelines[*OpenShift Pipelines Operator*], xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#serverless-install-web-console_installing-openshift-serverless[*OpenShift Serverless Operator*], and xref:../../virt/install/installing-virt.adoc#virt-subscribing-to-the-catalog_installing-virt[*Container-native virtualization*] Operator are installed, respectively.
```

Comment:
Fixed 1 and 2. 
Wrt: 
3. The Operator is part of the name of the Operator so again this is as per GUI and any GUI element has to be in bold. But will provide this as a feedback to the UX and UI team, this should be consistent for all Operators
4. Rechecked and retested, I know the docs: https://app_sbo_mods--ocpdocs.netlify.app/openshift-enterprise/latest/virt/install/installing-virt.html mention - OpenShift virtualization but there is not such operator on the Operator Hub. What I saw was Container-native virtualization operator. And when I installed it and deployed it, the VM card was displayed in the *Add* page as it should.

![Screenshot from 2020-06-23 22-53-28](https://user-images.githubusercontent.com/16647625/85435877-c22a3a00-b5a5-11ea-94e8-c78e1c04644a.png)
![Screenshot from 2020-06-23 22-53-19](https://user-images.githubusercontent.com/16647625/85435899-cb1b0b80-b5a5-11ea-9c11-6559d3cd3034.png)
![Screenshot from 2020-06-23 22-52-43](https://user-images.githubusercontent.com/16647625/85435903-cce4cf00-b5a5-11ea-8159-2ba02f336416.png)

Reply:
Cool, yeah the CNV part was just a question. If they update the operator name in the future we can always fix it here then.

As for the bold, I still don't think it's necessary, because you're not actually referring to them when looking at them in the GUI, you're just referencing the operator in general. But it's not a big deal.

However, the CNV "Operator" that you fixed to be inside the xref link isn't properly inside the bold.

Reply:
I did not make the "Operator" in CNV bold as it is not part of the name as per the GUI. So, now I have removed the bold from all the Operators, that way I don't have to refer to them by their actual name the way it is on the GUI, and I can say Operator only once without putting it in the link/making it bold. Hope this works.